vpdq/cpp/CMakeLists.txt

```diff
@@ -15,78 +15,84 @@ pkg_check_modules(LIBAV REQUIRED IMPORTED_TARGET
 
 set(CMAKE_CXX_STANDARD 14)
 
+set (PDQSOURCES
+    pdq/cpp/common/pdqhashtypes.cpp
+    pdq/cpp/hashing/pdqhashing.cpp
+    pdq/cpp/common/pdqhamming.cpp
+    pdq/cpp/io/hashio.cpp
+    pdq/cpp/downscaling/downscaling.cpp
+    pdq/cpp/hashing/torben.cpp
+)
+
+set (PDQHEADERS
+    pdq/cpp/common/pdqhashtypes.h
+    pdq/cpp/common/pdqbasetypes.h
+    pdq/cpp/common/pdqhamming.h
+    pdq/cpp/hashing/pdqhashing.h
+    pdq/cpp/io/hashio.h
+    pdq/cpp/downscaling/downscaling.h
+    pdq/cpp/hashing/torben.h
+)
+
 set(SOURCES
-    ../../pdq/cpp/common/pdqhashtypes.cpp
-    ../../pdq/cpp/hashing/pdqhashing.cpp
-    ../../pdq/cpp/common/pdqhamming.cpp
-    ../../pdq/cpp/io/hashio.cpp
-    ../../pdq/cpp/downscaling/downscaling.cpp
-    ../../pdq/cpp/hashing/torben.cpp
-    ./hashing/filehasher.cpp
-    ./hashing/bufferhasher.cpp
-    ./hashing/matchTwoHash.cpp
-    ./io/vpdqio.cpp
+    vpdq/cpp/hashing/bufferhasher.cpp
+    vpdq/cpp/hashing/filehasher.cpp
+    vpdq/cpp/hashing/matchTwoHash.cpp
+    vpdq/cpp/io/vpdqio.cpp
 )
 
 SET(HEADERS
-    ../../pdq/cpp/common/pdqbasetypes.h
-    ../../pdq/cpp/common/pdqhashtypes.h
-    ../../pdq/cpp/common/pdqhamming.h
-    ../../pdq/cpp/hashing/pdqhashing.h
-    ../../pdq/cpp/io/hashio.h
-    ../../pdq/cpp/downscaling/downscaling.h
-    ../../pdq/cpp/hashing/torben.h
-    ./hashing/filehasher.h
-    ./hashing/vpdqHashType.h
-    ./hashing/bufferhasher.h
-    ./hashing/matchTwoHash.h
-    ./io/vpdqio.h
+    vpdq/cpp/hashing/bufferhasher.h
+    vpdq/cpp/hashing/filehasher.h
+    vpdq/cpp/hashing/vpdqHashType.h
+    vpdq/cpp/hashing/matchTwoHash.h
+    vpdq/cpp/io/vpdqio.h
 )
 
-# TODO: Add more specific directories
-include_directories (
-    ../..
-)
+include_directories( vpdq/cpp/bin vpdq/cpp/hashing vpdq/cpp/io ${CMAKE_SOURCE_DIR} )
 
-add_library(vpdqlib
+add_library(vpdq
```

Comment:
Renamed `vpdqlib` to just `vpdq`. The output file before this was `libvpdqlib.a`, now it is just `libvpdq.a` like every other library is.